app/(afterLogin)/(planningAndReporting)/planning-and-reporting/_components/planning/index.tsx

```
@@ -277,6 +277,29 @@ function Planning() {
               loading={getPlanningLoading}
             >
               <div>
+                <Row className="flex justify-start mb-1 ">
+                  <div className="text-gray-400 ">
+                    {(() => {
+                      const planDate = dayjs(dataItem?.createdAt);
+                      const today = dayjs() || dayjs().subtract(1, 'day');
+                      const thisFriday = dayjs().day(5); // 0 = Sunday, ..., 5 = Friday
+                      const adjustedThisFriday = today.day() > 5 ? thisFriday.add(7, 'day') : thisFriday;
```

ESLint

[error] Insert `⏎·······················`

(prettier/prettier)